tests/freezer-spec.js

```diff
@@ -123,6 +123,38 @@ describe("Freezer test", function(){
 		assert.notEqual( updated, data );
 	});
 
+	it( "Set a value to null", function(){
+		data.set('a', null);
+
+		var updated = freezer.getData();
+
+		assert.deepEqual( updated.a, null );
+
+		assert.notEqual( updated, data );
+	});
+
+	it( "Update a numerical value to null", function(){
+		data.set('a', 1);
+		data.set('a', null);
+
+		var updated = freezer.getData();
+
+		assert.deepEqual( updated.a, null );
+
+		assert.notEqual( updated, data );
+	});
+
+	it( "Update a string value to null", function(){
+		data.set('a', 'blah');
+		data.set('a', null);
```

Comment:
This test has a failure just here. You are setting `a` in the frozen object to `'blah'`.
Then you are setting the attribute `a` of the same frozen object to `null`, but `data` is not in the freezer store anymore, so that change doesn't get reflected.

In this case I will `assert.equal(updated.a, 'blah')` to make the test pass. Or fix the test using:

```
	it( "Update a string value to null", function(){
		data = data.set('a', 'blah');
		data.set('a', null);

		var updated = freezer.getData();

		assert.deepEqual( updated.a, null );

		assert.notEqual( updated, data );
	});
```
The numeric example is passing just because you are setting the value `1` to an attribute that already has a value of `1`, so no update is triggered.